kv_ip_config.txt

```diff
@@ -0,0 +1 @@
+172.31.16.19 15900
```

Comment:
this file is generated by running test. remove it from PR.

Reply:
fixed